### PR TITLE
ACQ-752: fix the billing postcode component to allow hide completely

### PR DESCRIPTION
**Pull request by ekocian**
### Description
fix the billing postcode component to allow hide including its title label. Also include a modification on the debug bar to fill the form depending on the country

### Ticket
https://financialtimes.atlassian.net/browse/ACQ-752

### Screenshots
When it need to be shown
image
When it need to be hidden
image
Before fix
image

### Reminder
Have you completed these common tasks (remove those that don't apply)?

- [x] **Documentation** updated or created
- [x] **Tests** written for new or updated for existing functionality
- [x] **Stories** updated to use this change
- [ ] **Accessibility** checked for screen readers and contrast
- [ ] **Design Review** ran past the designer
- [ ] **Product Review** ran past the product owner

### CODE REVIEW

**Review by ststdimov — APPROVED** [review]
LGTM

File: components/billing-city.jsx [hunk 1]
```diff
@@ -2,12 +2,18 @@ import React from 'react';
...
 import classNames from 'classnames';
 
-export function BillingCity({
+export function BillingCity ({
 	hasError = false,
 	value = '',
 	isDisabled = false,
 	isHidden = false,
 }) {
+	const BillingCityFieldClassNames = classNames([
```
Comment (maguero) [inline components/billing-city.jsx:11]: nitpicking: can we please use lowercase for this const variables? `BillingCityFieldClassNames` => `billingCityFieldClassNames`

**Review by maguero — APPROVED** [review]
Nice fix on `debug` component 🎉 , it will help us to test quickly now.
I just left two nitpicking non blocking changes.

File: components/billing-city.jsx (hunk 1 above)
Comment (ekocian) [reply components/billing-city.jsx]: good catch! dd27ae625b8f731dea801e85bfd84f9ae0919017

File: components/debug.jsx [hunk 2]
```diff
@@ -49,18 +49,25 @@ export function Debug({ isTest = false, showHelpers = false, links = {} }) {
 	// This env var gets set in production. We use this when creating email addresses in case any
 	// get into production so Membership know who to come to about deleting them.
 	var SYSTEM_CODE = document.documentElement.getAttribute('data-next-app') || 'n-conversion-forms';
+	var COUNTRY_CODE = window.FT && window.FT.country || 'GBR';
+
+	var billingPostcodeByCountry = {
```
[the remark this answers is not in the document]
Comment (ekocian) [reply components/debug.jsx]: dd27ae625b8f731dea801e85bfd84f9ae0919017

**Review by ststdimov — APPROVED** [review]

**Review by maguero — APPROVED** [review]

**Pull request merged**